### ISSUE TITLE
Weird results when training on custom dataset

### ISSUE THREAD

**Issue by IvonaTau** [woozzu/tagan#18]
Hi, I got some weird looking generated images with patterns when training your model with default settings on custom dataset (dataset contains 6K images of people wearing clothes)

Could it be an issue with learning rate (or other hyperparameters) ? 

![gan_img_4_70](https://user-images.githubusercontent.com/14354389/71732629-08e57580-2e48-11ea-8be5-b953e9eee16f.png)

**Comment by IvonaTau** [comment]
OK, so it seems those artifacts are only present in earlier training stages. After training for a couple of hundred epochs they disappeared.
